ignite/templates/module/create/files/base/x/{{moduleName}}/client/cli/suite_test.go.plush

```diff
@@ -0,0 +1,54 @@
+package cli_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"<%= modulePath %>/testutil/network"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	locked bool
+	net    *network.Network
+	cfg    network.Config
+}
+
+func (s *IntegrationTestSuite) network() *network.Network {
+	s.net = network.New(s.T(), s.cfg)
+	s.locked = true
+	return s.net
+}
+
+func (s *IntegrationTestSuite) waitForNextBlock() {
+	s.T().Log("wait for next block")
+	s.Require().NoError(s.net.WaitForNextBlock())
+}
+
+func (s *IntegrationTestSuite) SetupTest() {
+	s.T().Log("setting up test")
+	s.cfg = network.DefaultConfig()
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	s.T().Log("setting up integration test suite")
+	s.cfg = network.DefaultConfig()
+}
```

Comment:
Not sure but wouldn't it make sense to set the config per test and remove the config initialization from the suite?

Reply:
We are setting the config per test, but since we have a global config we are resetting the config for each test